mindjunkies/lecture/__init__.py

```
@@ -0,0 +1,2 @@
+
+default_app_config = 'mindjunkies.lecture.apps.LectureConfig'
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)